### ISSUE TITLE
better support of multicoop setups

### ISSUE THREAD

**Issue by JuliusR** [foodcoops/sharedlists#22]
Currently, article synchronization cannot be restricted to a single instance in a multicoop setup. Is that correct?

Each foodcoop should be able to synchronize articles with its own private data source (FTP credentials from supplier, ...). That data source should not be available to other foodcoops.

**Comment by JuliusR** [comment]
Maybe we should not add this feature to [sharedlists](https://github.com/foodcoops/sharedlists), but proceed with foodcoops/foodsoft#396.

**Comment by wvengen** [comment]
Yes, I'd love to have sync to external datasources available from Foodsoft.
One source could be sharedlists, but logging into a supplier website with your own credentials would also be possible then - which would open the way for automated ordering on the supplier website when the order is closed.

**Comment by wvengen** [comment]
Perhaps creating a gem to interface with various suppliers would be useful here, to allow Foodsoft and Sharedlists to use it, as well open this up for use in other software.
